c10/core/ScalarType.h

```diff
@@ -320,6 +320,17 @@ static inline ScalarType toValueType(ScalarType t) {
   }
 }
 
+static inline ScalarType toComplexType(ScalarType t) {
+  switch (t) {
+    case ScalarType::Float:
+      return ScalarType::ComplexFloat;
+    case ScalarType::Double:
+      return ScalarType::ComplexDouble;
+    default:
+      AT_ERROR("Unknown Complex ScalarType");
```

Comment:
Isn't AT_ERROR deprecated? We should only use `TORCH_CHECK` now right?

Reply:
oh okay updated